fourkitchens/dropai
# feat: add service to extract text from drupal entities

Added an "EntityToText" service that will convert entities to text using OpenAI API endpoints.

Testing steps: 
1. Use the AI Inspect tab on either an image or an audio entity. Verify that the source displays:
- For an image, a description of the image given by the Vision endpoint
- For an audio file, a transcription of the file. Note that I only tested with a .wav file but it should work with flac, mp3, mp4, mpeg, mpga, m4a, ogg, wav, or webm.

Note:
Jim instructed me just to concentrate on the AI and not so much the architecture for now. It's not the prettiest but hopefully it works.
